### PR TITLE
Add ImageJ and AMPT installation instructions

### LINKED ISSUE #113: Add Install Instructions to User Guide
Idea came up during our install meeting with Holly to add install instructions at the end of the user guide.

### PR DESCRIPTION
**Pull request by APatrickJ**
Fixes #113

### CODE REVIEW

File: doc_source/docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Start the ImageJ Updater (_Help_ > _Update..._)
+
+![Start ImageJ Updater](img/start_imagej_updater.png)
+
+### Add a new update site
+
+Click on the _Manage update sites_ button to bring up the site management dialoag.
+
+Add AMPT as a new update site.
```
Comment (palbee) [inline doc_source/docs/installation.md:35]: We may want to rephrase this as "If AMPT is not in the list of update sites,  add the AMPT update site"...

**Review by palbee — CHANGES_REQUESTED** [review]
Since we're listed now in the update sites be default now we should probably note that the user may need to just activate AMPT.

File: doc_source/docs/installation.md (hunk 1 above)
Comment (APatrickJ) [reply doc_source/docs/installation.md]: Fair point. Updated in 9c1cf57.

**Review by palbee — APPROVED** [review]
Looks good.

**Comment by palbee** [comment]
Once this is merged, feel free to do a version bump and we'll do a release.
